modules/start/pages/managing-users.adoc

```diff
@@ -0,0 +1,41 @@
+= Managing Users in an Organization
+:experimental:
+
+When you set up a TigerGraph Cloud account, you are granted the privileges of an Organization Admin for your company.
+
+TigerGraph Cloud allows an Organization Admin to manage multiple users in your organization and control their access to various solutions.
+
+This page will explain the user access controls. For help with setting up solutions, visit xref:create-a-solution.adoc[].
+
+== Manage users in an organization
+The account that creates an organization is automatically given the privileges of an Organization Admin.
+Upon login, you have the opportunity to add additional users to your organization with different roles and privileges.
+
+=== Organization roles and privileges
+
+The privileges granted to the various roles are as follows:
+
+|====
+|Action | Organization Member | Organization Solution Admin |Organization Admin
+|Can add and remove new members | No 🚫 | No 🚫|  Yes✅
+|Can view organization billing information | No 🚫 |No 🚫|Yes✅
+|Can create new solutions |No 🚫|Yes ✅|Yes✅
+|Can view solution activities |No 🚫|Yes ✅|Yes✅
+|Can shut down/modify an existing solution |No 🚫|Yes ✅|Yes✅
+|Can view solution data |Only if assigned to the solution |Yes✅|Yes✅
+|Can access GraphStudio| Only if assigned to the solution |Only if assigned to the solution |Only if assigned to the solution
+|====
+
+=== Access Management
```

Comment:
Access management is a vague heading. I think most items on this page can have their own page under **[ User management ]** :  

* Invite new users
* Roles and privileges (you should write a short paragraph to explain the concept of roles and privileges in the context of TigerGraph cloud. 
* Grant roles to users
* Create teams
* Delete users
* Manage access to solutions

It's also important to distinguish between database users and TigerGraph Cloud users here.